tox.ini

```diff
@@ -57,6 +57,7 @@ commands =
 [testenv:pypy]
 basepython = pypy
 commands =
+    pip install PyPyDispatcher>=2.0.6
```

Comment:
If we start supporting pypy then I think PyPyDispatcher>=2.0.6 should be added to setup.py instead of tox.ini. 
It can be a bit tricky though: the best way is to use environment markers, but I'm still not sure how well are they supported. As I recall, they require pip > 6.0 and a recent setuptools, pip and setuptools used to be incompatible in evn markers format. But I've tried them a couple years ago; maybe it is better now. An alternative is to have a condition in setup.py, but if we add a condition there we won't be able to use universal wheels.